.github/workflows/node.js.yml

```diff
@@ -46,5 +46,12 @@ jobs:
       - name: Test unit
         run: yarn workspace esbuild-node-externals test
 
-      - name: Test examples (integration)
-        run: yarn workspace example-basic build
+      - name: Test integration esbuild@0.12
+        run: |
+          yarn workspace example-basic add esbuild@0.12
+          yarn workspace example-basic build
+
+      - name: Test integration esbuild@0.24
+        run: |
+          yarn workspace example-basic add esbuild@0.24
+          yarn workspace example-basic build
```

Comment:
I would remove the `yarn workspace example-basic add esbuild@0.24` so that the version in package.json can be used. So when we upgrade esbuild to eg 0.25 we don't need to update the CI file

Reply:
NP. Done